racetrack_client/racetrack_client/client/deploy.py

```diff
@@ -116,7 +116,8 @@ def send_deploy_request(
             job_url = _wait_for_deployment_result(lifecycle_url, deploy_id, user_auth, [])
         except Exception as e:
             raise DeploymentError(e)
-
+        if warnings := _get_warning(lifecycle_url, deploy_id, user_auth):
+            logger.warning(warnings)
```

Comment:
I think it would be better to move it inside `_wait_for_deployment_result`, so the warning can be printed out right after it appears in the database, without waiting for the deployment to finish.

Reply:
Original reasoning was to avoid extra logic, at the cost of a request. I'll move it inside `_wait_for_deployment_result`.